operator/api/v1/csiscaleoperator_types.go

```diff
@@ -387,11 +387,12 @@ func init() {
 type CSIReason string
 
 const (
-	CSIConfigured       CSIReason = "Successful"
+	CSIConfigured       CSIReason = "successful"
```

Comment:
Hi @nitishkumar4 , Why do we need to change these errors now ?

Reply:
These strings are updated in Status section, do you think other teams automation would depend on the string case used here? If so, then we should revert back to as it was for now.